webpack.config.js

```diff
@@ -54,6 +51,10 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
         },
         devtool: argv.mode === 'development' ? 'eval-source-map' : 'source-map',
         plugins: [
+            () => {
```

Comment:
I am not sure if plugins are written this way resp. if this has other unwanted side effects... Maybe we should use something like the [`WebpackShellPlugin`](https://github.com/1337programming/webpack-shell-plugin)? Or maybe there is an even better solution, since I can't believe we are the only people with this problem :confused: